script/binary

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
I'm curious - what platform are you targetting that does not have `bash` at `/bin/bash`, but _does_ have `env` at `/usr/bin/env`?

Reply:
It's not a matter of not having `/bin/bash` but using the right version.

Right now I am working on OSX. For some reason, as in I didn't explicitly did so, I have multiple version of bash. `bin/bash` is an older version (`3.2.57`) which doesn't some of the constructs in the script files. My shell is setup to use `/usr/local/bin/bash` which is a newer version (`4.4.5`).   

I wasn't obvious that was the problem at first sight. Using `/usr/bin/env` solved the problem.

Does it make sense?